### PR TITLE
[GS End User][W-8545419] Checklist Section 5

### PR DESCRIPTION
**Pull request by fparodi-salesforce**
# Critical Changes

# Changes
Create custom metadata to add new section 5  and his sub section in the End User GS page
# Issues Closed

# Community Ideas Delivered

# New Metadata
- Custom Metadata
  - GetStartedChecklistSection.Your_Data
  - GetStartedChecklistItem.View_Dashboard
  - GetStartedChecklistItem.Standard_Repor_Type
  - GetStartedChecklistItem.Prebuilt_NPSP_Report
<br/>

- Custom Label
  - gseuChecklistItemPrebuiltNPSPReportDesc
  - gseuChecklistItemPrebuiltNPSPReportLinkLabel
  - gseuChecklistItemPrebuiltNPSPReportPriBtnLabel
  - gseuChecklistItemPrebuiltNPSPReportSecBtnLabel
  - gseuChecklistItemPrebuiltNPSPReportTitle
  - gseuChecklistItemStandardReportTypeDesc
  - gseuChecklistItemStandardReportTypeLinkLabel
  - gseuChecklistItemStandardReportTypePriBtnLabel
  - gseuChecklistItemStandardReportTypeTitle
  - gseuChecklistItemViewDashboardDesc
  - gseuChecklistItemViewDashboardLinkLabel
  - gseuChecklistItemViewDashboardPriBtnLabel
  - gseuChecklistItemViewDashboardTitle
  - gseuChecklistYourDataDesc
  - gseuChecklistYourDataTitle
# Deleted Metadata

### CODE REVIEW

**Comment by fparodi-salesforce** [comment]
[W-8545419](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008kxODIAY/view)

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Title of checklist item FundraisingProcess in Get Started</shortDescription>
         <value>Visualize Your Fundraising Processes</value>
     </labels>
+    <labels>
+        <fullName>gseuChecklistItemPrebuiltNPSPReportDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item Prebuilt NPSP Report in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7336]: I recommend changing this to: Description of Prebuilt NPSP Report checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>NPSP includes 87 of the most commonly-used constituent and fundraising reports. Use these reports as is, or customize them.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemPrebuiltNPSPReportLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist item Prebuilt NPSP Report in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7344]: I recommend changing this to: Link label of Prebuilt NPSP Report checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 3]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Learn more about NPSP Reports in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemPrebuiltNPSPReportPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Prebuilt NPSP Report in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7352]: I recommend changing this to: Button label of Prebuilt NPSP Report checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 4]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Run Reports</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemPrebuiltNPSPReportSecBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Prebuilt NPSP Report in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7360]: I recommend changing this to: Button label of Prebuilt NPSP Report checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 5]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Reports Workbook</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemPrebuiltNPSPReportTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Prebuilt NPSP Report in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7368]: I recommend changing this to: Title of Prebuilt NPSP Report checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 6]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Run a Prebuilt NPSP Report</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item Standard Report Type in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7376]: I recommend changing this to: Description of Standard Report Type checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 7]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>The Lightning Report Builder gives you the ability to create your own custom reports using standard report types. The report type you choose determines the objects and fields available in your report.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist item Standard Report Type in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7384]: I recommend changing this to: Link label of Standard Report Type checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 8]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Learn more about Standard Report Types in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypePriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Standard Report Type in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7392]: I recommend changing this to: Button label of Standard Report Type checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 9]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Create Reports</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Standard Report Type in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7400]: I recommend changing this to: Title of Standard Report Type checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 10]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Create a Report from a Standard Report Type</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item View Dashboard in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7408]: I recommend changing this to: Description of View Dashboard checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 11]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Dashboards pull data from one of more reports and display it side by side in the same window. NPSP includes three dashboards that you can use as is or customize.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist item View Dashboard in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7416]: I recommend changing this to: Link label of View Dashboard checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 12]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Learn more about Dashboards in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item View Dashboard in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7424]: I recommend changing this to: Button label of View Dashboard checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 13]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>View Dashboards</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item View Dashboard in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7432]: I recommend changing this to: Title of View Dashboard checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 14]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>View a Dashboard</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistYourDataDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Your Data in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7440]: I recommend changing this to: Description of Understand Your Data checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 15]
```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Reports and Dashboards help organize your data so that you can tell the story of your organization&apos;s impact. NPSP includes out-of-the-box reports, or you can create your own.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistYourDataTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Your Data in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7448]: I recommend changing this to: Title of Understand Your Data checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels (hunk 10 above)
Comment (smarovich) [reply src/labels/CustomLabels.labels:7408]: @fparodi-salesforce I think you might have missed this comment.

**Review by smarovich — APPROVED** [review]
Labels look good! There is one small comment to resolve.

**Review by force2b — APPROVED** [review]

**Pull request merged**